dasharo-stability/capsule-update.robot

```diff
@@ -104,6 +105,25 @@ CUP150.001 Capsule Update
     Should Contain    ${out}    CapsuleMax
     Should Not Contain    ${out}    CapsuleLast
 
+CUP150.002 Capsule Update Progress Bar - Dasharo Logo
```

Comment:
This must be done after `CUP190.001`, and preferably after a gap in numbering (e.g. as `CUP250` or more) to allow for adding new tests in between if such a need arises later.

Reply:
[Done](https://github.com/Dasharo/open-source-firmware-validation/pull/520/commits/af7dcc6dbc6f01376fd395e21c92c4908925eadb#diff-e5457f1e95f583456d0728805e1a4d16e63924b5c7f79a4ff0bbdcfed0751b7bR159)